x/docs/md/help/executable.md

```diff
@@ -28,10 +28,10 @@ If the API key is incorrect, then re-run the `configure` command:
 exercism configure --key=YOUR_API_KEY
 ```
 
-Next look at the `Exercises Directory`. The current directory which is being used to fetch exercises can be viewed using the following command:
+Next look at the Exercises Directory. It will look something like this:
 
-```bash
-exercism configure --dir
+```plain
+Exercises Directory: /home/you/exercism
```

Comment:
But then we might as well do that for both, right? Change `exercism debug` to `exercism configure` and then look for `--key=` for the first section and `--dir=` for the second. No need to mention `exercism debug` on this page at all?